### ISSUE TITLE
invalid syntax cross_entropy = -tf.reduce_sum(labels * tf.log(softmax), reduction_indices=[1]))

### ISSUE THREAD

**Issue by jiankang1991** [MarvinTeichmann/tensorflow-fcn#4]
I think in the line 47 of `loss.py`. there is an invalid syntax mistake:`-tf.reduce_sum(labels * tf.log(softmax), reduction_indices=[1]))` the last bracket should be removed.

**Comment by MarvinTeichmann** [comment]
Thanks! Properly a copy-paste error.
